src/main/java/gov/nasa/jpl/mbee/mdk/ems/MMSUtils.java

```diff
@@ -172,8 +172,8 @@ public static ObjectNode getServerElementsRecursively(Project project, String el
         if (depth > 0) {
             requestUri.setParameter("depth", java.lang.Integer.toString(depth));
         }
-        else {
-            requestUri.setParameter("recurse", java.lang.Boolean.toString(recurse));
+        else if (recurse) {
+            requestUri.setParameter("depth", java.lang.Integer.toString(9999));
```

Comment:
I don't like that we're specifying an arbitrarily large depth. We should use -1. I believe all negative numbers are treated as infinite depth.